runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java

```diff
@@ -594,8 +604,27 @@ Color getHidden(String item, int gePrice, int haPrice, boolean isTradeable)
 			: null;
 	}
 
-	Color getItemColor(Color highlighted, Color hidden)
+	Color getProfitable(int gePrice, int haPrice)
 	{
+		final int FIRE_RUNE_PRICE = itemManager.getItemPrice(FIRE_RUNE);
```

Comment:
most people use a staff for alching so not accounting for fire rune price is more correct imo